### PR TITLE
HNPS Basalt to TitanSMA SM recorder swap

### PR DESCRIPTION
**Pull request by Thomas-Benson**
- Removing Basalt (552) from HNPS LC 21
- Replacing with TitanSMA (001709) at same location
- Using latest TitanSMA firmware version 4.5.22
- New recorder has orientation of 0 (true North)
- Streams affected HNPS LC 21 HN-12Z

### CODE REVIEW

File: install/recorders.csv [hunk 1]
```diff
@@ -1432,6 +1432,7 @@ Nanometrics,Titan,TitanSMA,001261,BKZ,21,0,,0,0,2021-10-22T00:32:00Z,9999-01-01T
 Nanometrics,Titan,TitanSMA,001705,APZ,21,0,,0,0,2022-04-13T03:37:00Z,9999-01-01T00:00:00Z
 Nanometrics,Titan,TitanSMA,001706,RLNS,20,0,,0,0,2022-02-01T03:16:00Z,9999-01-01T00:00:00Z
 Nanometrics,Titan,TitanSMA,001708,GVZ,20,0,,0,0,2021-11-25T01:47:00Z,9999-01-01T00:00:00Z
+Nanometrics,Titan,TitanSMA,001709,HNPS,21,192,,0,0,2024-01-09T00:14:00Z,9999-01-01T00:00:00Z
```
Comment (CBurton90) [inline install/recorders.csv:1435]: Just needs to be `0` for azi?

Comment (mnaguit) [reply install/recorders.csv:1435]: If azimuth is updated to 0 (from 192), streams should be updated to HN-ENZ (from HN-12Z)?

Comment (ozym) [reply install/recorders.csv:1435]: I expect this to only for national network sites. Freefield SM should be consistent to allow easy queries.

Comment (CBurton90) [reply install/recorders.csv:1435]: @mnaguit @ozym I think we discussed this problem in depth within delta but alas our decision is buried in an equipment change ticket of which I cant remember 😆 . 

I thought we agreed on:

- National Network and regional strong motion (hut FBA recorder and vault/cabinet FBA sensors where skyview is readily available to orient north) = HNZ | N | E

- Urban strong motion (I believe this is just called the strong motion network, grey box FBA and MEMS recorders) where typically there is no skyview within the building and recorder is aligned with a building structure with an arbitrary azimuth i.e. a wall = HNZ | 1 | 2. There will be edge cases where that arbitrary azimuth of the building structure is aligned to true north (like this site) but we still label the channels as axial for network consistency.

Comment (Thomas-Benson) [reply install/recorders.csv]: Updated orientation. Apparently I'd changed it to 0 locally but forgot to push it to github before I finished yesterday 🤣

**Review by salichon — APPROVED** [review]
great thx

File: install/recorders.csv (hunk 1 above)
Comment (CBurton90) [reply install/recorders.csv]: @staylorofford and I worked on a brief document towards the end of last year. Hopefully it clears up what our processes are regarding strong motion orientation and metadata. Keen to get any further input you may have, we will ideally find a nice accessible home for it. 

cc @ozym @mnaguit @Thomas-Benson @salichon 

[Strong_Motion_Instrument_Orientation_Guide_CB.docx](https://github.com/GeoNet/delta/files/14104407/Strong_Motion_Instrument_Orientation_Guide_CB.docx)

**Review by CBurton90 — APPROVED** [review]

**Pull request merged**